toolchain-msvc.cmake

```diff
@@ -39,6 +39,8 @@ if(USE_CLANG_CL)
         set(CMAKE_C_COMPILER_TARGET "arm64-pc-windows-msvc")
         set(CMAKE_CXX_COMPILER_TARGET "arm64-pc-windows-msvc")
     else()
+        set(CMAKE_C_FLAGS "-m32")
+        set(CMAKE_CXX_FLAGS "-m32")
```

Comment:
Shouldn't it be also added to `elseif(ARCH STREQUAL "arm")` case above? Or maybe there could be some generic check for 32-bit arch?

Reply:
I guess it can be added there as `clang-cl --print-supported-cpus --target=arm-pc-windows-msvc -m64` list is way shorter compared to the same command but with `-m32`. Let's see.